main.css

```diff
@@ -0,0 +1,70 @@
+*{
+    padding: 0;
+    margin: 0;
+}
+.container{
+    display: flex;
+    background-color: darkorange;
+    padding: 15px;
+    box-shadow: 0px 3px whitesmoke;
+    border-bottom-left-radius: 20px;
+}
+.logo{
+    font-size: 55px;
+    margin-left: 45px;
+}
+.a{
+    justify-content: flex-end;
+    display: flex;
+    align-items: center;
+    margin-left: auto;
+}
+.a1{
```

Comment:
use a better class name that gives it meaning. currently, I'm not able to understand the meaning of a1.